warehouse/cli/sponsors.py

```diff
@@ -0,0 +1,592 @@
+      url="https://www.datadoghq.com/",
+      image="datadog.png",
+      activity=[
+        "PyPI uses Datadog to collect metrics from the applications, services, and infrastructure behind the scenes allowing for the team to measure the impact of new changes, monitor for problems, and alert when systems fail."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=True,
+      one_time=False,
+      sidebar=False,
+    ),
+    dict(
+      name="Fastly",
+      service="CDN",
+      url="https://www.fastly.com/",
+      image="fastly.png",
+      activity=[
+        "PyPI uses Fastly's CDN to quickly serve content to end-users, allowing us to minimize our hosting infrastructure and obscure possible downtime."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=True,
+      one_time=False,
+      sidebar=False,
+    ),
+    dict(
+      name="DigiCert",
+      service="EV certificate",
+      url="https://www.digicert.com/",
+      image="digicert.png",
+      activity=[
+        "PyPI uses Digicert to secure communication and prove identity with an EV Certificate."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=True,
+      one_time=False,
+      sidebar=False,
+    ),
+    dict(
+      name="StatusPage",
+      service="Status page",
+      url="https://statuspage.io",
+      image="statuspage.png",
+      activity=[
+        "The PyPI team uses StatusPage to communicate downtime, service degradation, or maintenance windows to end-users."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=True,
+      one_time=False,
+      sidebar=False,
+    ),
+    dict(
+      name="Mozilla",
+      url="https://www.mozilla.org",
+      image="mozilla.png",
+      activity=[
+        "In late 2017 the Python Software Foundation was awarded a Mozilla Open Source Support (MOSS) award.",
+        "This award was used to move PyPI from its legacy codebase and deploy Warehouse - the new codebase powering PyPI."
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=False,
+      one_time=True,
+      sidebar=False,
+    ),
+    dict(
+      name="Open Technology Fund",
+      url="https://www.opentech.fund",
+      image="otf.png",
+      activity=[
+        "In 2019 the Python Software Foundation was awarded a contract through the OTF Core Infrastructure Fund.",
+        "This contract included an audit and improvements to the accessibility of PyPI, internationalization and translations for PyPI user interface, and security features including two-factor authentication and API tokens for uploads.",
+      ],
+      footer=False,
+      psf_sponsor=False,
+      infra_sponsor=False,
+      one_time=True,
+      sidebar=False,
+    ),
+]
+
+
+@warehouse.group()  # pragma: no branch
+def sponsors():
+    """
+    Manage operations on top of sponsors.
+    """
+
+
+@sponsors.command()
+@click.pass_obj
+def populate_db(config):
```

Comment:
I wasn't sure if a click command was the best option here. From my django background, I'd rather go with a data migration, but I couldn't figure out how to do so with sqlalchemy yet. Please, let me know what you think about this.